src/ScheduledTasks/Client/DefaultScheduleClient.cs

```diff
@@ -0,0 +1,257 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using Microsoft.DurableTask.Client;
+using Microsoft.DurableTask.Client.Entities;
+using Microsoft.DurableTask.Entities;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.DurableTask.ScheduledTasks;
+
+/// <summary>
+/// Represents a handle to a scheduled task, providing operations for managing the schedule.
+/// </summary>
+// TODO: Isolate system entity from user entities
```

Comment:
What's the plan for isolating system entities from user entities? Do we have a work item tracking this?